serverless.yml

```diff
@@ -9,42 +9,52 @@ plugins:
 
 custom:
   stackName: ${self:service}-${self:provider.stage}
-  xray: "Active" # AWS Xray tracing of requests, queries
+  xray: ''  # AWS Xray tracing of requests, queries. Set to 'Active' to actively trace.
   db:
-    dbname: "TEMPLATE"
+    dbname: TEMPLATE
     minCapacity: 2
     maxCapacity: 4
     secondsUntilAutoPause: 43200 # if not stage=='prd'
   layers:
     psycopg: "arn:aws:lambda:${self:provider.region}:898466741470:layer:psycopg2-py37:1"
-    default:
-      - ${self:custom.layers.psycopg}
+    default: []
+      # - ${self:custom.layers.psycopg}
   secrets:
     prefix: "${self:service}/${self:provider.stage}"
     rds: "${self:custom.secrets.prefix}/rds/credentials"
-  vpc:
-    securityGroupIds:
-      - "Fn::GetAtt": ServerlessSecurityGroup.GroupId
-    subnetIds:
-      - Ref: ServerlessSubnetA
-      - Ref: ServerlessSubnetB
-      - Ref: ServerlessSubnetC
+  network:
+    classB: 10 # second octet of VPC CIDR block (10.classB.0.0/16)
+  bastion:  # EC2 host to access database
+    sshKeyPairName: ''  # set this to enable bastion ec2 instance
+    alertEmails: 'me@mydomain.xyz'  # get SNS notifications of alarms at this address
+    banner: https://ops.jetbridge.com/public/banner.txt
+    bootstrap: https://ops.jetbridge.com/public/bastion_bootstrap.sh
+    instanceType: t3a.nano
+    tenancy: default
+    bannerEnabled: 'true'
+    tcpForwardingEnabled: 'true'
+    x11ForwardingEnabled: 'false'
+    logRetentionInDays: 731
```

Comment:
2 years? A bit long and expensive IMO.

Reply:
I have no idea what it costs, but this should be very small